joomla-extensions/weblinks
# Move weblinks stats to weblinks repo

Pull Request for Issue #288 (redo because of merge conflicts)

### Summary of Changes

Move web links statistics shown on the admin control panel from Joomla core to a system plugin in the web links package.

### Testing Instructions

- install 3.7.0 (as this is a new plugin event)
- Create at least one published web link.
- apply this pr
- discover the new plugin
- install ist
- Enable the system plugin.
- You should now have an entry about the weblinks count.

### Expected result

You should now have an entry about the weblinks count.

### Actual result

You don't see any count in the CMS it selfe (in 3.7.0b)

### Documentation Changes Required

None.
